src/visibility-and-privacy.md

```diff
@@ -107,15 +107,22 @@ mod crate_helper_module {
 // crates linking against this one.
 pub fn public_api() {}
 
+//This is by default private, and it isn't available to external crates.
+const rate_limit:i32 = 32;
+
 // Similarly to 'public_api', this module is public so external crates may look
 // inside of it.
 pub mod submodule {
-    use crate_helper_module;
-
+    use super::crate_helper_module;
+    use crate::rate_limit;
+    use super::public_api;
+    
     pub fn my_method() {
         // Any item in the local crate may invoke the helper module's public
         // interface through a combination of the two rules above.
         crate_helper_module::crate_helper();
+        public_api();
+        let doube_limit = rate_limit * 2;
```

Comment:
I'm not sure this really belongs here.  This example (especially this `my_method`) is here to illustrate the combination of the two rules above.  I'm not sure how this relates to those rules?

Reply:
rate_limit is private only to the file itself and can't be accessed outside of it.  i added it as an example of that fact to contrast with the note about public_api()